webapp/views/layout.jade

```diff
@@ -13,6 +13,9 @@ html(lang='en', ng-app='lunchr')
         script(src='bower_components/angular-ui-router/release/angular-ui-router.js')
         script(src='javascripts/angular-app.js')
         script(src='javascripts/controllers.js')
+        script(src='bower_components/lodash/dist/lodash.js')
```

Comment:
I am not 100% sure, I can look into if we need it or not. When I installed the framework lodash came with it. I think the map depends on it.